ara_plumes/models.py

```diff
@@ -780,3 +762,61 @@ def background_subtract(
         clean_vid[i] = cv2.subtract(frame, background_img_np)
 
     return clean_vid
+
+
+def flatten_edge_points(mean_points: PlumePoints, vari_points: PlumePoints) -> Float2D:
+    """
+    Convert points on center and edge to distances from each other for a given frame.
+    NOTE: if no pair can be form, points are rejected. If more than one candidate
+    pair is found, the first pair is selected.
+    """
+    pairs = _create_radius_pairs(mean_points, vari_points)
+    rad_dist = []
+    for radius, center_points, edge_points in pairs:
+        dist = np.linalg.norm(center_points - edge_points)
+        rad_dist.append((radius, dist))
+    return np.array(rad_dist)
+
+
+def _create_radius_pairs(
+    mean_points: PlumePoints, vari_points: PlumePoints
```

Comment:
This is generic to two curves, correct?  Maybe it makes sense to name the variables generically `curve_1_pts` and `curve_2_pts`